docset/windows/appx/Get-NonRemovableAppsPolicy.md

```diff
@@ -0,0 +1,169 @@
+---
+author: romaclac
+description: Use this topic to help prevent the uninstall of specific Windows apps with Windows PowerShell.
+external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-help.xml
+keywords: powershell, cmdlet, msix, appx, block uninstall
+manager: jvintzel
+Module Name: Appx
+ms.assetid: 
+ms.author: romaclac
+ms.date: 02/05/2020
+ms.mktglfcycl: manage
+ms.prod: w10
+ms.reviewer:
+ms.sitesec: library
+ms.technology: powershell-windows
+ms.topic: reference
+online version:
+schema: 2.0.0
+title: Get-NonRemovableAppsPolicy
+---
+
+# Get-NonRemovableAppsPolicy
+
+## SYNOPSIS
+Gets a list of the app packages that are installed and configured as non-removable apps.
+
+## SYNTAX
```

Comment:
When there is more than one parameter sets, they are named, for example ByPath or ByOnline, an example https://raw.githubusercontent.com/MicrosoftDocs/office-docs-powershell/master/sharepoint/sharepoint-ps/sharepoint-online/Get-SPOUser.md

Reply:
I believe I've updated this one correctly.